[2025-02-25T23:11:40.062+0530] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection_processing_dag.process_input_file manual__1740504989 [queued]>
[2025-02-25T23:11:40.065+0530] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection_processing_dag.process_input_file manual__1740504989 [queued]>
[2025-02-25T23:11:40.065+0530] {taskinstance.py:1361} INFO - Starting attempt 2 of 4
[2025-02-25T23:11:40.069+0530] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): process_input_file> on 2025-02-25 17:36:29.772408+00:00
[2025-02-25T23:11:40.074+0530] {standard_task_runner.py:57} INFO - Started process 92562 to run task
[2025-02-25T23:11:40.079+0530] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'fraud_detection_processing_dag', 'process_input_file', 'manual__1740504989', '--job-id', '430', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline_dag.py', '--cfg-path', '/var/folders/kz/_l5jq2xx6db29k_3xyv2j8n40000gp/T/tmpqy2g4wdt']
[2025-02-25T23:11:40.080+0530] {standard_task_runner.py:85} INFO - Job 430: Subtask process_input_file
[2025-02-25T23:11:40.103+0530] {task_command.py:416} INFO - Running <TaskInstance: fraud_detection_processing_dag.process_input_file manual__1740504989 [running]> on host ninja-its-macbook-air-90.local
[2025-02-25T23:11:40.127+0530] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='fraud_detection_processing_dag' AIRFLOW_CTX_TASK_ID='process_input_file' AIRFLOW_CTX_EXECUTION_DATE='2025-02-25T17:36:29.772408+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__1740504989'
[2025-02-25T23:11:40.128+0530] {data_pipeline_dag.py:55} INFO - Received kwargs: {'conf': <airflow.configuration.AirflowConfigParser object at 0x1051fe9d0>, 'dag': <DAG: fraud_detection_processing_dag>, 'dag_run': <DagRun fraud_detection_processing_dag @ 2025-02-25 17:36:29.772408+00:00: manual__1740504989, state:running, queued_at: 2025-02-25 17:36:29.776499+00:00. externally triggered: True>, 'data_interval_end': DateTime(2025, 2, 25, 17, 36, 29, 772408, tzinfo=Timezone('UTC')), 'data_interval_start': DateTime(2025, 2, 25, 17, 36, 29, 772408, tzinfo=Timezone('UTC')), 'ds': '2025-02-25', 'ds_nodash': '20250225', 'execution_date': <Proxy at 0x17dea8d40 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x16b8538b0>, 'execution_date', DateTime(2025, 2, 25, 17, 36, 29, 772408, tzinfo=Timezone('UTC')))>, 'expanded_ti_count': None, 'inlets': [], 'logical_date': DateTime(2025, 2, 25, 17, 36, 29, 772408, tzinfo=Timezone('UTC')), 'macros': <module 'airflow.macros' from '/Users/nc25577_suryateja/Documents/air_pro/venv/lib/python3.9/site-packages/airflow/macros/__init__.py'>, 'next_ds': <Proxy at 0x16b89b2c0 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x16b8538b0>, 'next_ds', '2025-02-25')>, 'next_ds_nodash': <Proxy at 0x16b89bd80 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x16b8538b0>, 'next_ds_nodash', '20250225')>, 'next_execution_date': <Proxy at 0x17deb2b80 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x16b8538b0>, 'next_execution_date', DateTime(2025, 2, 25, 17, 36, 29, 772408, tzinfo=Timezone('UTC')))>, 'outlets': [], 'params': {'file_path': '/Users/nc25577_suryateja/Documents/air_pro/input_files/user_2385861_report_4.json'}, 'prev_data_interval_start_success': DateTime(2025, 2, 25, 17, 32, 49, 993335, tzinfo=Timezone('UTC')), 'prev_data_interval_end_success': DateTime(2025, 2, 25, 17, 32, 49, 993335, tzinfo=Timezone('UTC')), 'prev_ds': <Proxy at 0x16b8abd00 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x16b8538b0>, 'prev_ds', '2025-02-25')>, 'prev_ds_nodash': <Proxy at 0x16b8ab6c0 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x16b8538b0>, 'prev_ds_nodash', '20250225')>, 'prev_execution_date': <Proxy at 0x16b8ab440 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x16b8538b0>, 'prev_execution_date', DateTime(2025, 2, 25, 17, 36, 29, 772408, tzinfo=Timezone('UTC')))>, 'prev_execution_date_success': <Proxy at 0x16b8ab980 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x16b8538b0>, 'prev_execution_date_success', DateTime(2025, 2, 25, 17, 32, 49, 993335, tzinfo=Timezone('UTC')))>, 'prev_start_date_success': DateTime(2025, 2, 25, 17, 32, 50, 255578, tzinfo=Timezone('UTC')), 'run_id': 'manual__1740504989', 'task': <Task(PythonOperator): process_input_file>, 'task_instance': <TaskInstance: fraud_detection_processing_dag.process_input_file manual__1740504989 [running]>, 'task_instance_key_str': 'fraud_detection_processing_dag__process_input_file__20250225', 'test_mode': False, 'ti': <TaskInstance: fraud_detection_processing_dag.process_input_file manual__1740504989 [running]>, 'tomorrow_ds': <Proxy at 0x16b8abac0 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x16b8538b0>, 'tomorrow_ds', '2025-02-26')>, 'tomorrow_ds_nodash': <Proxy at 0x16b8aba40 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x16b8538b0>, 'tomorrow_ds_nodash', '20250226')>, 'triggering_dataset_events': <Proxy at 0x16b858bc0 with factory <function TaskInstance.get_template_context.<locals>.get_triggering_events at 0x16b853790>>, 'ts': '2025-02-25T17:36:29.772408+00:00', 'ts_nodash': '20250225T173629', 'ts_nodash_with_tz': '20250225T173629.772408+0000', 'var': {'json': None, 'value': None}, 'conn': None, 'yesterday_ds': <Proxy at 0x16b8aba00 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x16b8538b0>, 'yesterday_ds', '2025-02-24')>, 'yesterday_ds_nodash': <Proxy at 0x16b8abdc0 with factory functools.partial(<function lazy_mapping_from_context.<locals>._deprecated_proxy_factory at 0x16b8538b0>, 'yesterday_ds_nodash', '20250224')>, 'templates_dict': None}
[2025-02-25T23:11:40.128+0530] {data_pipeline_dag.py:59} INFO - Conf type: <class 'airflow.configuration.AirflowConfigParser'>, value: <airflow.configuration.AirflowConfigParser object at 0x1051fe9d0>
[2025-02-25T23:11:40.128+0530] {data_pipeline_dag.py:70} INFO - Processing input file at /Users/nc25577_suryateja/Documents/air_pro/input_files/user_2385861_report_4.json
[2025-02-25T23:11:40.128+0530] {data_pipeline_dag.py:73} ERROR - File not found: /Users/nc25577_suryateja/Documents/air_pro/input_files/user_2385861_report_4.json
[2025-02-25T23:11:40.129+0530] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/Users/nc25577_suryateja/Documents/air_pro/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/Users/nc25577_suryateja/Documents/air_pro/venv/lib/python3.9/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/Users/nc25577_suryateja/Documents/air_pro/airflow/dags/data_pipeline_dag.py", line 74, in process_input_file
    raise FileNotFoundError(f"File not found: {file_path}")
FileNotFoundError: File not found: /Users/nc25577_suryateja/Documents/air_pro/input_files/user_2385861_report_4.json
[2025-02-25T23:11:40.147+0530] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=fraud_detection_processing_dag, task_id=process_input_file, execution_date=20250225T173629, start_date=20250225T174140, end_date=20250225T174140
[2025-02-25T23:11:40.151+0530] {standard_task_runner.py:104} ERROR - Failed to execute job 430 for task process_input_file (File not found: /Users/nc25577_suryateja/Documents/air_pro/input_files/user_2385861_report_4.json; 92562)
[2025-02-25T23:11:40.174+0530] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-02-25T23:11:40.185+0530] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
